### DISCUSSION TITLE
Package for Ubuntu

### DISCUSSION THREAD

**Discussion by fabwu** [Netatalk/netatalk#434] [Ideas]
Do you plan to update your package for Ubuntu?

The [current package](https://launchpad.net/ubuntu/+source/netatalk) on launchpad is outdated and doesn't seem to have any supporters.

**Comment by jonassmedegaard** [comment]
I guess this question is better asked at your distribution.

Most packages in Ubuntu are really done in Debian. For quite some years I did that task in Debian, but unfortunately I can no longer do that in my spare time (hint hint: if you can help raise money for my work then please get in touch).

Development status for netatalk in Debian is here: https://tracker.debian.org/pkg/netatalk

**Comment by rdmark** [comment]
@jonassmedegaard Thanks for maintaining the Debian package all these years! If someone would volunteer to take up the mantle from you, what are the steps required to become a Debian package maintainer, if I may ask?

FWIW I put up a PR today to merge your downstream typo fixes. https://github.com/Netatalk/Netatalk/pull/218

**Comment by jonassmedegaard** [comment]
Quoting Daniel Markstedt (2023-02-20 22:19:04)
> @jonassmedegaard Thanks for maintaining the Debian package all these years! If someone would volunteer to take up the mantle from you, what are the steps required to become a Debian package maintainer, if I may ask?

Join the Netatalk team by subscribing to the mailinglist at
https://alioth-lists.debian.net/cgi-bin/mailman/listinfo/pkg-netatalk-devel
and then post a mailinglist to that same list just to say hi - then we
can maintain the package together (I am happy to help if not alone).

Then if you still want to be able to maintain packages on your own in
Debian (which I certainly encourage you to do, but which is not
necessary for team-maintained packages) then see more details of that
process at https://www.debian.org/devel/join/

> FWIW I put up a PR today to merge your downstream typo fixes. https://github.com/Netatalk/Netatalk/pull/218

Nice!

**Comment by rdmark** [comment]
Joined and sent! Co-maintaining the package sounds great. This is all new to me so please bear with me getting up to speed.

**Comment by jonassmedegaard** [comment]
Quoting Daniel Markstedt (2023-02-21 06:24:57)
> Joined and sent! Co-maintaining the package sounds great. This is all new to me so please bear with me getting up to speed.

Great!

Don't worry, we will figure it out together.
